### DISCUSSION TITLE
Download a .qmd from a quarto website

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#8665] [Q&A]
### Description

Is it possible to allow the download of a .qmd file in a quarto website?

I am thinking in a download button within the html file based on the .qmd file.

**Comment by mcanouil** [comment]
See <https://quarto.org/docs/output-formats/html-code.html#code-tools>.

Edit: Depending on your setup you can also use regular link `[A (hidden not rendered) Quarto document](path/to/a/quarto/_document.qmd)`.

**Comment by ManuelSpinola** [comment]
Thank you very much.
